test/parallel/test-child-process-spawn-shell.js

```diff
@@ -48,6 +48,26 @@ command.on('close', common.mustCall((code, signal) => {
   assert.strictEqual(commandOutput.trim(), 'bar');
 }));
 
+// Verify that shell features can not be passed through
```

Comment:
The rest of the file isn't like this, but can I suggest adding curly braces around this new block of code. That will isolate it from the rest of the file in a block scope.

Reply:
Yep. Should I add a comment to specify why, or should I instead wrap the rest of the test cases also? Looks like the other test cases are not sharing any state or data between them so perhaps wrap them too for consistency?

Reply:
I don't think you need to add any comments about why the block scopes are used. Comments describing what's going on inside those block scopes are always welcome though 😄

Reply:
Right :-)
I meant if that only case would be isolated than the rest, but yeah, I guess current comments make sense then.